db/migrate/20160413135658_create_topic_section.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTopicSection < ActiveRecord::Migration
+  def change
+    create_table :topic_sections do |t|
+      t.integer :topic_id, null: false
+      t.string :title, null: false
+      t.string :description, null: false
```

Comment:
I 10% remember that Luke asked if he could create sections without giving a title and description - and he could. I have this feeling we might have some sections without headings..